### PR TITLE
Fix coordinate conversion for BEDPE files

### PR DESCRIPTION
**Pull request by clintval**
I made a mistake and thought the coordinate system used here was 0-based.

It is 1-based! I've also added this clarification to the docs so others will benefit.

### CODE REVIEW

**Review by ameynert — APPROVED** [review]
Looks good to me!

File: src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala [hunk 1]
```diff
@@ -230,12 +230,12 @@ object BreakpointCategory extends Enumeration {
...
- * @param left_min_pos          Minimum coordinate of left breakends
- * @param left_max_pos          Maximum coordinate of left breakends
+ * @param left_min_pos          Minimum coordinate of left breakends (1-based)
+ * @param left_max_pos          Maximum coordinate of left breakends (1-based)
  * @param left_strand           Strand at left breakends
  * @param right_contig          Contig name for right side of breakpoint
- * @param right_min_pos         Minimum coordinate of right breakends
- * @param right_max_pos         Maximum coordinate of right breakends
+ * @param right_min_pos         Minimum coordinate of right breakends (1-based)
+ * @param right_max_pos         Maximum coordinate of right breakends (1-based)
```
Comment (nh13) [inline src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala:238]: do we want to say inclusive here, or is that implied?

**Review by nh13 — COMMENTED** [review]
qq about the coordinate system being 1-based inclusive.  Perhaps we should say that somewhere prominent (readme?)

File: src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala (hunk 1 above)
Comment (clintval) [reply src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala:238]: Can't hurt, I'll add it!

**Comment by clintval** [comment]
> qq about the coordinate system being 1-based inclusive. Perhaps we should say that somewhere prominent (readme?)

Good idea. I'll add this as as sub-task to https://github.com/fulcrumgenomics/fgsv/pull/34

**Pull request merged**